### PR TITLE
More detailed exception messages in `ListenerConfigurationChecker`

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #31

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # Yii Event Change Log
 
-
 ## 1.0.2 under development
 
-- no changes in this release.
+- Enh #32: More detailed exception messages in `ListenerConfigurationChecker` (vjik)
```
Comment (samdark) [inline CHANGELOG.md:5]:
```suggestion
- Enh #32: Use more detailed exception messages in `ListenerConfigurationChecker` (vjik)
```

File: src/ListenerConfigurationChecker.php [hunk 2]
```diff
@@ -74,6 +75,26 @@ public function check(array $configuration): void
...
+     */
+    private function createNotCallableMessage($definition): string
+    {
+        if (is_array($definition)
+            && array_keys($definition) === [0, 1]
+            && is_string($definition[1])
+        ) {
+            if (is_string($definition[0]) && class_exists($definition[0])) {
+                return $definition[0] . ' could not instantiate or method "' .
+                    $definition[1] . '" not exists in him.';
```
Comment (samdark) [inline src/ListenerConfigurationChecker.php:89]:
```suggestion
                return sprintf('Could not instantiate "%s" or "%s" method is not defined in this class.', $definition[0], $definition[1]);
```

File: src/ListenerConfigurationChecker.php [hunk 3]
```diff
@@ -74,6 +75,26 @@ public function check(array $configuration): void
...
+            }
+            if (is_object($definition[0])) {
+                return 'Method "' . $definition[1] . '" not exists in class ' . get_class($definition[0]) . '.';
```
Comment (samdark) [inline src/ListenerConfigurationChecker.php:92]:
```suggestion
                return sprintf('"%s" method is not defined in "%s" class.', $definition[1], get_class($definition[0]));
```

**Review by samdark — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (vjik) [reply CHANGELOG.md]: Done

File: src/ListenerConfigurationChecker.php (hunk 2 above)
Comment (vjik) [reply src/ListenerConfigurationChecker.php]: Done

File: src/ListenerConfigurationChecker.php (hunk 3 above)
Comment (vjik) [reply src/ListenerConfigurationChecker.php]: Done

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**